src/actions/hiro-fetch.ts

```diff
@@ -10,6 +10,7 @@ async function hiroFetch(url: string, options: RequestInit = {}) {
   return fetch(url, {
     ...options,
     headers: { ...options.headers, ...apiHeaders },
+    cache: "no-store",
```

Comment:
What does this do? Does it instruct the Hiro API to not store the results?

Reply:
it instructs next not to cache the results